### PR TITLE
b2-mid-mod-KB

### PR DESCRIPTION
**Pull request by Kathybui732**
Kathy bui

### CODE REVIEW

**Comment by Kathybui732** [comment]
Kathy Bui

I got through all 3 user stories

File: app/controllers/ride_mechanics_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class RideMechanicsController < ApplicationController
+
+  def create
+    if Ride.exists?(id: params[:ride_id])
```
Comment (megstang) [inline app/controllers/ride_mechanics_controller.rb:4]: Nice, I like this edge case that you took care of here.

File: app/views/mechanics/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<% @mechanic.rides.each do |ride| %>
+<%= ride.name %>
+<% end %>
+
+<%= form_tag "/mechanics/#{@mechanic.id}/ride_mechanics", method: :post do %>
+<%= label_tag :ride_id %>
+<%= text_field_tag :ride_id %>
+
+<%= submit_tag "Submit" %>
+<% end %>
```
Comment (megstang) [inline app/views/mechanics/show.html.erb:13]: 👍 good job here

File: app/views/parks/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<%= @park.name %>
+<%= @park.admission_price %>
+
+<% @park.rides.alpha_order.each do |ride| %>
+<%= ride.name %>
+<% end %>
+
+<%= @park.rides.average_rating %>
```
Comment (megstang) [inline app/views/parks/show.html.erb:8]: good job calling out to these model methods -- your delegation of logic within this MVC pattern looks good 👍

**Pull request closed without merging**